patch-package
--- a/node_modules/passport-saml/lib/passport-saml/saml.js
+++ b/node_modules/passport-saml/lib/passport-saml/saml.js
@@ -714,6 +714,18 @@ SAML.prototype.processValidlySignedAssertion = function(xml, inResponseTo, callb
       if (authnStatement[0].$ && authnStatement[0].$.SessionIndex) {
         profile.sessionIndex = authnStatement[0].$.SessionIndex;
       }
+
+      if (authnStatement[0].AuthnContext) {
+        var authnContext = authnStatement[0].AuthnContext;
+        if (authnContext) {
+          if (authnContext[0] && authnContext[0].AuthnContextClassRef) {
+            var authnContextClassRef = authnContext[0].AuthnContextClassRef;
+            if (authnContextClassRef) {
+              profile.authnContextClassRef = authnContextClassRef[0];
+            }
+          }
+        }
+      }
     }
 
     var subject = assertion.Subject;
