patch-package
--- a/node_modules/spid-passport/index.js
+++ b/node_modules/spid-passport/index.js
@@ -375,8 +375,7 @@ const generateAuthorizeRequest = function(req, samlClient, callback) {
       if (samlClient.options.identifierFormat) {
         request["samlp:AuthnRequest"]["samlp:NameIDPolicy"] = {
           "@xmlns:samlp": "urn:oasis:names:tc:SAML:2.0:protocol",
-          "@Format": samlClient.options.identifierFormat,
-          "@AllowCreate": "true"
+          "@Format": samlClient.options.identifierFormat
         };
       }
 
